webrtc.html

```diff
@@ -4414,37 +4414,43 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           <dt>checking</dt>
 
           <dd>The <code><a>RTCIceTransport</a></code> has received at least one
-          remote candidate, and is checking candidate pairs but has not yet
-          found a connection. In addition to checking, it may also still be
-          gathering.</dd>
+          remote candidate, and either is checking candidate pairs but has not yet
+          found a connection, or consent checks [[!RFC7675]] have failed on all
+          previously successful candidate pairs.  In addition to checking, it
+          may also still be gathering.</dd>
 
           <dt>connected</dt>
 
           <dd>The <code><a>RTCIceTransport</a></code> has found a usable
           connection, but is still checking other candidate pairs to see if
           there is a better connection. It may also still be gathering and/or
-          waiting for additional remote candidates.</dd>
+          waiting for additional remote candidates. If consent checks [[!RFC7675]]
+          fail on the connection in use, and there are no other successful
+          candidate pairs available, then the state transitions to "disconnected".</dd>
```

Comment:
I guess it could transition to "checking" as well, depending on whether or not the transport is currently checking.